src/title-dropdown/title-dropdown.component.html

```diff
@@ -1,7 +1,9 @@
-<div dropdown>
-  <button class="btn title-dropdown-toggle" dropdownToggle>
+<div dropdown [ngClass]="{disableCursorClass: schema && schema['x_editor_disabled']}">
+  <div [ngClass]="{disableEventsClass: schema && schema['x_editor_disabled']}">
+    <button class="btn title-dropdown-toggle" dropdownToggle>
```

Comment:
@harunurhan It worked but for the title-dropdown when you on hover it changes the color but the cursor is not-allowed. Check it again from the latest commit.